### PR TITLE
maintainers guide

### PR DESCRIPTION
**Pull request by 1ucian0**
maintainers guide

### CODE REVIEW

File: docs/maintainers_guide.rst [hunk 1]
```diff
@@ -0,0 +1,198 @@
+
+#################
+Maintainers Guide
+#################
+
+This document calls *mainteiner* a contributor with merging privilages.
```
Comment (javabster) [inline docs/maintainers_guide.rst]:
```suggestion
This document defines a *maintainer* as a contributor with merge privileges.
```

File: docs/maintainers_guide.rst [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+The task described here are mostly related with the Qiskit release process and with internal processes.
```
Comment (1ucian0) [inline docs/maintainers_guide.rst]:
```suggestion
The tasks described here are mostly related with the Qiskit release process and with internal processes.
```

File: docs/maintainers_guide.rst [hunk 3]
```diff
@@ -0,0 +1,198 @@
+
+#################
+Maintainers Guide
+#################
+
+This document calls *mainteiner* a contributor with merging privilages.
+The tasks described here are mostly related with the Qiskit release process and with internal processes.
```
Comment (javabster) [inline docs/maintainers_guide.rst]:
```suggestion
The information detailed here is mostly related to Qiskit releases and other internal processes.
```

File: docs/maintainers_guide.rst [hunk 4]
```diff
@@ -0,0 +1,198 @@
+
+#################
+Maintainers Guide
+#################
+
+This document defines a *maintainer* as a contributor with merging privileges.
```
Comment (javabster) [inline docs/maintainers_guide.rst]:
```suggestion
This document defines a *maintainer* as a contributor with merge privileges.
```

**Review by javabster — APPROVED** [review]
LGTM 🚀

**Pull request merged**